src/components/navigation/nav.css

```diff
@@ -1,22 +1,30 @@
 header {
   margin-top: 0;
+  margin-bottom: 2rem;
   width: 100vw;
   height: 5rem;
   display: flex;
   align-items: center;
   justify-content: space-around;
-  background-color: #000;
-  color: #fff;
+  background-color: #fff;
+  border-bottom: 1px solid rgb(205, 205, 205);
   font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
 }
 
-button {
-  margin: 0 1rem 0 1rem;
-  background-color: transparent;
+.h1 {
+  color: #0290ff;
+  font-size: 2.5rem;
+  font-weight: 900;
+}
+
+.button-2 {
   border: none;
+  background-color: #fff;
+  margin-left: 2.5rem;
 }
 
-.button {
+.nav-button {
+  color: #000;
+  font-size: 1.5rem;
   text-decoration: none;
```

Comment:
- [ ] Kindly make sure that all the elements in the header are in the correct position, add the login icon, and use the styles assigned pay attention to the differences in font color to match with the Zeplin design. 👇🏽 

| Zeplin | Your design |
|---|---|
| ![image](https://github.com/Booze33/book-store/assets/58642949/8a618978-4460-419a-a791-7eeb3be18297) | ![image](https://github.com/Booze33/book-store/assets/58642949/cb67bcd3-66e7-44e9-b376-89021f7dabb4) |